### PR TITLE
Update blocklist.yml with subdomain

### PR DESCRIPTION
**Pull request by phantomdon**

### CODE REVIEW

**Review by binsytest — COMMENTED** [review]
- url: "www.binanceblockchainweek.com"

File: whitelist.yaml
Comment (VALPASSPORT) [inline whitelist.yaml]: https://pocketoption.com/

**Review by ITGuruRu — COMMENTED** [review]
- url: putyourfuckingtextonthefuckingblockchain.com

File: blocklist.yaml [hunk 1]
```diff
@@ -2225,3 +2225,4 @@
   - url: bonkticket.xyz
   - url: orca-foundation.org
```
Comment (ITGuruRu) [inline blocklist.yaml]: - url: putyourfuckingtextonthefuckingblockchain.com

**Pull request merged**